mod_form.php

```diff
@@ -753,9 +753,6 @@ protected function add_max_file_size_field() {
                                  get_string('maximumsizelabel', 'coursework'));*/
         $moodleform->addHelpButton('maxbytes', 'maximumsize', 'mod_coursework');
         $moodleform->hideif('maxbytes', 'use_turnitin', 'eq', '1');
-
```

Comment:
Unrelated changes?
Ok to have in the PR but would be neater in a separate commit.